src/sentry/integrations/client.py

```diff
@@ -114,11 +114,14 @@ def json(self):
         return self
 
 
-def track_response_code(host, code):
-    metrics.incr('integrations.http_response', tags={
-        'host': host,
-        'status': code
-    })
+def track_response_code(integration, code):
+    metrics.incr(
+        'integrations.http_response',
+        sample_rate=1.0,
+        tags={
+            'integration': integration,
```

Comment:
@markstory: I changed this tag from `host` to `integration` because `host` is a protected tag that we apply at the machine-level.

Reply:
Sounds good. I'll update the related dashboads/monitors when this ships.